docs/content/preview/explore/fault-tolerance/transaction-availability.md

```diff
@@ -0,0 +1,234 @@
+yugabyted stop --base_dir=/tmp/ybd2
+```
+
+List the nodes via 
+```sh
+yb-admin list_tablets ysql.yugabyte txndemo
+```
+
+```output.sh
+Tablet-UUID                        Key Range         Leader-IP       Leader-UUID
+7e2dfb66a4654aa5b2fb133b446aaabc   [0x0000, 0x5554]  127.0.0.3:9100  7402fbc9c6384d80bb7bcd09b89dbca9
+a9b4675fdaaa4d4b949adc5e53d183bf   [0x5555, 0xAAA9]  127.0.0.3:9100  7402fbc9c6384d80bb7bcd09b89dbca9
+a8c50129f63642459a02ed4ee492a1f3   [0x0000, 0x5554]  127.0.0.1:9100  6789e52b1c334844a66078fe9fdf95fa
+```
+Notice that `127.0.0.2` is gone from the list and `127.0.0.3` is now the leader for the tablet that was in `127.0.0.2`(`7e2dfb66a..`)
+
+Commit the transaction.
+```output.sql
+yugabyte@yugabyte=# COMMIT;
+COMMIT
+Time: 6.243 ms
+```    
+We can see that even though the node `127.0.0.2` failed after receiving the provisional write, the transaction has succeeded.
+
+## Failure Scenario 2
+In this example, you are going to see how a transaction successfully completes when the node that is about to receive a provisional write fails.
+
+List the nodes via 
+```sh
+yb-admin list_tablets ysql.yugabyte txndemo
+```
+```output.sh
+Tablet-UUID                        Key Range         Leader-IP       Leader-UUID
+7e2dfb66a4654aa5b2fb133b446aaabc   [0x0000, 0x5554]  127.0.0.2:9100  4739b43f76184e1cab003b88686df290
+a9b4675fdaaa4d4b949adc5e53d183bf   [0x5555, 0xAAA9]  127.0.0.3:9100  7402fbc9c6384d80bb7bcd09b89dbca9
+a8c50129f63642459a02ed4ee492a1f3   [0x0000, 0x5554]  127.0.0.1:9100  6789e52b1c334844a66078fe9fdf95fa
+```    
+Connect to `127.0.0.3` and start a simple transaction to update the value of row `k1=1` to `30`.
+```sql
+ysqlsh -h 127.0.0.3  -U yugabyte -d yugabyte
+yugabyte@yugabyte=# BEGIN;
+BEGIN
+Time: 2.047 ms
+```    
+
+Stop the node:`127.0.0.2`. _Note_: We are stopping `127.0.0.2` as that is the node that contains `k1=1` which we figured from the steps in the [Identifying the key location](#identifying-the-key-location) section.
+```sh
+yugabyted stop --base_dir=/tmp/ybd2
+```
+
+List the nodes to verify node:`127.0.0.2` is gone.
+```sh
+yb-admin list_tablets ysql.yugabyte txndemo
+```
+```output.sh
+Tablet-UUID                        Key Range         Leader-IP       Leader-UUID
+7e2dfb66a4654aa5b2fb133b446aaabc   [0x0000, 0x5554]  127.0.0.3:9100  7402fbc9c6384d80bb7bcd09b89dbca9
+a9b4675fdaaa4d4b949adc5e53d183bf   [0x5555, 0xAAA9]  127.0.0.3:9100  7402fbc9c6384d80bb7bcd09b89dbca9
+a8c50129f63642459a02ed4ee492a1f3   [0x0000, 0x5554]  127.0.0.1:9100  6789e52b1c334844a66078fe9fdf95fa
+```   
+Commit the transaction
+```output.sql
+yugabyte@yugabyte=# UPDATE txndemo set k2=30 where k1=1; COMMIT;
+UPDATE 1
+Time: 1728.246 ms (00:01.728)
+COMMIT
+Time: 2.964 ms
+```    
+You can see that even though the node `127.0.0.2` failed before receiving the provisional write, the transaction has succeeded. This is because a new leader (node:`127.0.0.3`) was quickly elected.
+    
+## Failure of Transaction Manager(Coordinator)
```

Comment:
do we have precedent for calling this "Transaction Manager"? i thought we call it "Transaction Coordinator" everywhere? let's stay consistent if so

Reply:
In the current 2.17 docs...

Transaction coordinator: 14 results in 8 files.
Transaction manager: 12 results in 7 files.

So really, we can call it whatever we want. One global search and replace will sort it out.

Reply:
This [diagram](https://docs.yugabyte.com/preview/architecture/transactions/transactional-io-path/#:~:text=The%20following%20diagram%20depicts%20the%20high%2Dlevel%20steps%20of%20a%20distributed%20write%2Donly%20transaction%2C%20not%20including%20any%20conflict%20resolution%3A)  says `Transaction Manager` . Not sure how many other images show the concept. Let discuss this soon and make a call.

Reply:
Lets discuss with @rahuldesirazu, @robertsami offline & decide on the naming convention . Lets keep that change separate from this diff.